### DISCUSSION TITLE
Github Pages use Readme

### DISCUSSION THREAD

**Discussion by Nollknolle** [community/community#23125] [Other Feature Feedback, Questions, & Ideas]
<p>hey i have in github pages a readme.<br>
Thats in the file.</p>
<pre><code class="lang-auto"># TEST
</code></pre>
<p>but github adds an header<br>
<img src="https://user-images.githubusercontent.com/62543404/181074968-f06a7c4e-e9bd-4120-a4ec-b900f637b668.png" alt="grafik" data-base62-sha1="AjuuwKd8VUhKod81v7jZDt8iGx1" width="436" height="164"></p>
<p>can i remove that?<br>
thanks for your help :smiley:</p>

**Comment by jfullstackdev** [comment]
<p>where is the repo, I’m looking at your Repos tab I cannot see that one</p>

**Comment by Nollknolle** [comment]
<p>Hey,<br>
the repo is private but pages public.</p>
<p>Here is the URL: <a href="https://nollknolle.github.io/Spielerfreunde_Bot_v2/" class="inline-onebox" rel="noopener nofollow ugc">TEST | Spielerfreunde_Bot_v2</a><br>
RAW File: <a href="https://raw.githubusercontent.com/Nollknolle/Spielerfreunde_Bot_v2/main/docs/README.md?token=GHSAT0AAAAAABQF3PGTES2KSBQTYZHSN3N6YVMIV6A" rel="noopener nofollow ugc">https://raw.githubusercontent.com/Nollknolle/Spielerfreunde_Bot_v2/main/docs/README.md?token=GHSAT0AAAAAABQF3PGTES2KSBQTYZHSN3N6YVMIV6A</a></p>

**Comment by jfullstackdev** [comment]
<p>there are two headers at the source,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/f/cff3cfa093c90e7000653379352a0cd4c60d8a45.png" data-download-href="/uploads/short-url/tFDcltzC57hFLXxAYIkwWMgYXQx.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074975-3bad2a62-cb0b-4796-9c62-47e6323fb568.png" alt="image" data-base62-sha1="tFDcltzC57hFLXxAYIkwWMgYXQx" width="690" height="119" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/f/cff3cfa093c90e7000653379352a0cd4c60d8a45_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">758×131 4.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>the output is correct</p>

**Comment by Nollknolle** [comment]
<p>I see.<br>
Then GitHub adds the header by itself.<br>
Is there any way to remove the header on a Readme File?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s because of the different default themes of GitHub Pages for README,</p>
<p>I created a test, as you can see it’s the same as yours,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/a/9a0cea26b594fe0cc06e5f3ac8dde46048b18cd7.png" data-download-href="/uploads/short-url/lYNcoFtOkOrV9WIALouy3DAygiH.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074978-385b2259-3a85-4eb2-a580-e86f0b51956c.png" alt="image" data-base62-sha1="lYNcoFtOkOrV9WIALouy3DAygiH" width="662" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a0cea26b594fe0cc06e5f3ac8dde46048b18cd7_2_662x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a0cea26b594fe0cc06e5f3ac8dde46048b18cd7_2_993x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/a/9a0cea26b594fe0cc06e5f3ac8dde46048b18cd7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/a/9a0cea26b594fe0cc06e5f3ac8dde46048b18cd7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1139×860 18.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>now if I will add a theme, it looks like this</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/8/e84759cb4ebcd6567190626700856a9e80d1d7a1.jpeg" data-download-href="/uploads/short-url/x8PDLGV602ej5nlZozlKhZzAbQJ.jpeg?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074979-b64c23a2-ce8f-4bdb-8be2-1994a8bb7b1e.jpeg" alt="image" data-base62-sha1="x8PDLGV602ej5nlZozlKhZzAbQJ" width="662" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e84759cb4ebcd6567190626700856a9e80d1d7a1_2_662x500.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e84759cb4ebcd6567190626700856a9e80d1d7a1_2_993x750.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/8/e84759cb4ebcd6567190626700856a9e80d1d7a1.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/8/e84759cb4ebcd6567190626700856a9e80d1d7a1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1139×860 100 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>so yes that’s the default behavior,</p>
<p>and about removing it, maybe this one</p>
<aside class="onebox stackexchange" data-onebox-src="https://stackoverflow.com/questions/46375765/how-do-you-remove-header-on-github-pages">
  <header class="source">

      <a href="https://stackoverflow.com/questions/46375765/how-do-you-remove-header-on-github-pages" target="_blank" rel="noopener">stackoverflow.com</a>
  </header>

  <article class="onebox-body">
      <a href="https://stackoverflow.com/users/6542896/willy-david-jr" target="_blank" rel="noopener">
    <img alt="Willy David Jr" src="https://user-images.githubusercontent.com/47092464/181074981-0196d7f3-a637-401f-a3fe-2320e26c4302.jpeg" class="thumbnail onebox-avatar" width="256" height="256">
  </a>

<h4>
  <a href="https://stackoverflow.com/questions/46375765/how-do-you-remove-header-on-github-pages" target="_blank" rel="noopener">How do you remove header on Github Pages?</a>
</h4>

<div class="tags">
  <strong>github, jekyll, github-pages</strong>
</div>

<div class="date">
  asked by
  
  <a href="https://stackoverflow.com/users/6542896/willy-david-jr" target="_blank" rel="noopener">
    Willy David Jr
  </a>
  on <a href="https://stackoverflow.com/questions/46375765/how-do-you-remove-header-on-github-pages" target="_blank" rel="noopener">03:34AM - 23 Sep 17 UTC</a>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>but for sure, you have the full freedom to actually use HTML, CSS and JavaScript, enjoy !</p>

**Comment by Nollknolle** [comment]
<p>hey thanks,<br>
i created the file under:<br>
/docs/assets/css/style.scss</p>
<p>is that right? because it wont work. do i create it from the root directory?</p>
<p><img src="https://user-images.githubusercontent.com/62543404/181074982-4b9e09bd-9ee3-41c6-9f3c-1bf82627e620.png" alt="grafik" data-base62-sha1="r1Pq69OlyKgYCuzY8W8hK3wbEiU" width="575" height="438"><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/e/1e77abda85256c160c8c9a988a0ef1ae2192223f.png" data-download-href="/uploads/short-url/4lwIyFPudDp3oBTksqkpChQVvmn.png?dl=1" title="grafik" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/62543404/181074983-ec69ded3-7243-48e7-b6bc-a27899dc9974.png" alt="grafik" data-base62-sha1="4lwIyFPudDp3oBTksqkpChQVvmn" width="690" height="247" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/e/1e77abda85256c160c8c9a988a0ef1ae2192223f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">grafik</span><span class="informations">960×345 16.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
<img src="https://user-images.githubusercontent.com/62543404/181074984-86879970-4c02-4689-a7f0-c30ff40bd005.png" alt="grafik" data-base62-sha1="a2d6IkHObKUzRqqMOy3FxHDQCaH" width="573" height="404">
